requirements.txt

```diff
@@ -13,5 +13,8 @@ dataclasses; python_version == "3.6"
 dimod>=0.10.5
 numpy>=1.16
 
+# for solver mocks
+dwave-networkx>=0.8.9
```

Comment:
Does this new dependency deserve a comment in reno?

Reply:
I don't think it does. It's just an optional dependency (like dimod). And we use it just for testing.

Reply:
We say that to install from source, users should do `pip install -r requirements.txt` and that now has `dwave-networkx` (as well as dimod). How would users know it's optional?

Reply:
Because it's in package's `mocks` extra. Like dimod is in `bqm`/`dqm` extra. (In setup.py, and declared in package metadata).

Reply:
Perfect, thanks!